client/state/signup/optional-dependencies/reducer.js

```diff
@@ -9,7 +9,7 @@ import { suggestedUsernameSchema } from './schema';
 
 const suggestedUsername = createReducer( '',
 	{
-		[ SIGNUP_OPTIONAL_DEPENDENCY_SUGGESTED_USERNAME_SET ]: ( state = null, action ) => {
```

Comment:
Note that i needed to remove this initialization for the `no-unused-vars` not to fail.